docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.layout.scss

```diff
@@ -1,5 +1,7 @@
 @mixin hb-page-width {
-  max-width: hb-calculate-rems($hb-layout-max-width);
-  margin: 0 auto;
-  width: calc(100% - (2 * #{hb-calculate-rems($hb-gutter-width)}));
+  // scss-lint:disable ImportantRule
```

Comment:
weird!

Reply:
(that this has to happen now)

Reply:
Yeah, I'm not sure why the masthead styles need to be overridden but not in the global footer or brand bar give the source order but this is the only way it would work.